vm/src/builtins/builtinfunc.rs

```diff
@@ -142,6 +143,15 @@ impl PyBuiltinFunction {
     fn repr(&self) -> String {
         format!("<built-in function {}>", self.value.name)
     }
+    #[pyproperty(magic)]
+    fn text_signature(&self, vm: &VirtualMachine) -> PyObjectRef {
```

Comment:
```suggestion
    fn text_signature(&self, vm: &VirtualMachine) -> Option<String> {
```
With this signature, `None` will turn into Python `None` and `Some(String)` will turn into Python `str`

Reply:
I tried to apply it in d12d286 commit. Could you check it again?